cmd/main.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017 Intel Corporation
```

Comment:
How about to move it to 'cmd/multus/main.go' to put another binary in the repo...